utils/flare.py

```diff
@@ -88,8 +89,8 @@ def __init__(self, cmdline=False, case_id=None):
     def check_user_rights():
         if Platform.is_unix() and not os.geteuid() == 0:
             log.warning("You are not root, some information won't be collected")
-            choice = raw_input('Are you sure you want to continue [y/N]? ').lower()
-            if choice not in ['yes', 'y']:
+            choice = raw_input('Are you sure you want to continue [y/N]? ')
+            if "".join(choice.lower().split()) not in ['yes', 'y']:
```

Comment:
I'd say if someone typed `y e s` he probably had one too many :beers: and need to hold off sending his logs :wink: